lang/pl/zoom.php

```diff
@@ -0,0 +1,169 @@
+<?php
+// This file is part of the Zoom plugin for Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+
+/**
+ * Polish strings for zoom.
+ *
+ * @package    mod_zoom
+ * @copyright  2015 UC Regents
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+defined('MOODLE_INTERNAL') || die();
+
+$string['actions'] = 'Dziłania';
+$string['addtocalendar'] = 'Dodać do kalendarza';
+$string['alternative_hosts'] = 'Alternatywne Hosty';
+$string['alternative_hosts_help'] = 'Opcja alternatywnego hosta pozwala planować spotkania i desygnować drugiego Pro usera na tym że koncie żeby zacząć spotkanie albo webinar, jeżeli nie możecie. Użytkowniki otrzymają email powiadamiając ich że oni byli dodani jako alternatywni hosty, z linkiem dla początku spotkania. Rozdzielać wielu emeilów przecinkiem (bez spacji).';
+$string['allmeetings'] = 'Wszystkie spotkania';
+$string['apikey'] = 'Kluć Zoom API';
+$string['apikey_desc'] = '';
+$string['apisecret'] = 'Secret Zoom API';
+$string['apisecret_desc'] = '';
+$string['apiurl'] = 'Link do Zoom API';
+$string['apiurl_desc'] = '';
+$string['audio_both'] = 'VoIP i Telephonia';
+$string['audio_telephony'] = 'Tylko telephonia';
+$string['audio_voip'] = 'Tylko VoIP';
+$string['cachedef_zoomid'] = 'ID usera Zoom';
+$string['cachedef_sessions'] = 'Informacja-raport z zoom o userach';
+$string['calendardescriptionURL'] = 'Link dla połączenia: {$a}.';
+$string['calendardescriptionintro'] = "\Opis:\n{\$a}";
```

Comment:
Hi @mkikets99 

Should this be?
```suggestion
$string['calendardescriptionintro'] = "\nOpis:\n{\$a}";
```

Reply:
Yes, sorry...
Missed that...